src/helpers/cauldron/cauldron.js

```diff
@@ -1,22 +1,22 @@
 export const cook = async (contract, cookData, value) => {
-    const estimateGas = await contract.estimateGas.cook(
-      cookData.events,
-      cookData.values,
-      cookData.datas,
-      {
-        value,
-      }
-    );
+    // const estimateGas = await contract.estimateGas.cook(
+    //   cookData.events,
+    //   cookData.values,
+    //   cookData.datas,
+    //   {
+    //     value,
+    //   }
+    // );
   
-    const gasLimit = estimateGas.add(1000);
+    // const gasLimit = estimateGas.add(1000);
   
     const tx = await contract.cook(
       cookData.events,
       cookData.values,
       cookData.datas,
       {
         value,
-        gasLimit,
+        gasLimit: 500000000,
```

Comment:
Where is this coming from?
Why don't you trust the gas estimation?
Or will this just stay for testing?

Reply:
we should not use hard coded gas limits, pls respond on this @0xhardrefresh

Reply:
just for test, to check failed tx